homestar-runtime/Cargo.toml

```diff
@@ -53,14 +53,17 @@ ipfs-api = { version = "0.17", optional = true }
 ipfs-api-backend-hyper = { version = "0.6", features = ["with-builder", "with-send-sync"], optional = true }
 itertools = "0.10"
 libipld = { workspace = true }
-libp2p = { version = "0.51", default-features = false, features = ["kad", "request-response", "macros", "identify", "mdns", "gossipsub", "tokio", "dns", "mplex", "tcp", "noise", "yamux", "websocket"] }
+libp2p = { version = "0.51", default-features = false, features = ["kad", "request-response", "macros", "identify", "mdns", "gossipsub", "tokio", "dns", "mplex", "tcp", "noise", "yamux", "websocket", "ed25519", "secp256k1"] }
 libsqlite3-sys = { version = "0.26", features = ["bundled"] }
 openssl = { version = "0.10", features = ["vendored"] }
 proptest = { version = "1.2", optional = true }
+rand = "0.8.5"
 reqwest = { version = "0.11", features = ["json"] }
+sec1 = { version = "0.7.2", features = ["pem", "der"] }
 semver = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_ipld_dagcbor = { workspace = true }
+serde_with = { version = "*", features = ["base64"] }
```

Comment:
minor: I brought in serde_with in my PR too, we should version this vs `*`.

Reply:
this being left in was a mistake sorry